### PR TITLE
Bento template service POC

### PR DESCRIPTION
**Pull request by dmanek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "jss": "10.8.2",
     "jss-preset-default": "10.8.2",
     "moment": "2.29.1",
+    "mustache": "4.2.0",
```
Comment (alanorozco) [inline package.json:30]: We don't need the package listed since we keep a copy in `third_party/`

Comment (dmanek) [reply package.json:30]: @dethstrobe @alanorozco and I had this conversation offline. The current copy is `third_party` was committed ~3 yrs ago, so it does warrant an update. However, that'll be a separate task (tracked in #37505).

**Comment by Gg860ren** [comment]
ok

**Comment by Gg860ren** [comment]
okok

**Pull request closed without merging**